sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/AnalyzeTextOptions.java

```diff
@@ -201,6 +204,20 @@ public LexicalTokenizerName getTokenizerName() {
      * @param tokenFilters the tokenFilters value to set.
      * @return the AnalyzeRequest object itself.
      */
+    public AnalyzeTextOptions setTokenFilters(TokenFilterName... tokenFilters) {
```

Comment:
The options are input only

Reply:
We can leave the varargs setter here and remove the list one.

Reply:
Removed